src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -725,12 +725,20 @@
                                   IsChecked="{Binding ShowTabsAndSpacesInScriptEditor, Mode=TwoWay}"
                                   Header="{x:Static p:Resources.DynamoViewSettingsMenuShowTabsAndSpacesInScriptEditor}"/>
                         <MenuItem Focusable="False"
-                                  Header="{x:Static p:Resources.DynamoViewSettingsDefaultPythonEngine}">
-                            <ComboBox ItemsSource="{Binding DefaultPythonEngineOptions}"
-                                      SelectedValue="{Binding DefaultPythonEngine, Mode=TwoWay}"
-                                      SelectedValuePath="Value"
-                                      DisplayMemberPath="Name"
-                                      Style="{StaticResource SComboBox}"/>
+                                  Header="{x:Static p:Resources.DynamoViewSettingsDefaultPythonEngine}"
+                                  Name="PythonEngine">
+                            <MenuItem Focusable="False"
+                                      Header="{x:Static p:Resources.DefaultPythonEngineNone}"
+                                      IsCheckable="True"
+                                      IsChecked="{Binding PreferenceSettings.DefaultPythonEngine,Converter={StaticResource PythonSelectionToStringConverter},ConverterParameter='', Mode=TwoWay}"/>
```

Comment:
Since you already have a `DefaultPythonEngine` in `DynamoViewModel` I would suggest you bind to that instead.

Reply:
@mmisol Let me look at the code again. I was meant to ask you do we still need that property?

Reply:
I saw that was the pattern we were using for other settings. In my opinion it feels cleaner than binding to `PreferenceSettings` directly.

Reply:
Thanks, will change